utils/check_inits.py

```diff
@@ -249,7 +249,7 @@ def get_transformers_submodules():
             if fname == "__init__.py":
                 continue
             short_path = str((Path(path) / fname).relative_to(PATH_TO_TRANSFORMERS))
-            submodule = short_path.replace(os.path.sep, ".").replace(".py", "")
+            submodule = short_path.replace(".py", "").replace(os.path.sep, ".")
```

Comment:
Out of curiosity, why is this change necessary?

Reply:
Not anymore. That was when I had utils/py_utils.py but since it's moved to utils/generic.py, it's not a problem. It may become a problem later for other folks who name a file starting with /py tho.

Reply:
Good to keep then!